org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java

```diff
@@ -108,6 +108,7 @@ public void visitEnd() {
 	public void analyzeClass(final ClassReader reader) {
 		final ClassVisitor visitor = createAnalyzingVisitor(
 				CRC64.checksum(reader.b), reader.getClassName());
+		// TODO maybe we can SKIP_FRAMES here?
```

Comment:
I agree in the comment but separate issue? On the other hand I'm not sure what the benefit would be as frames are ignored at analysis time.